### PR TITLE
task28-Create-static-page

### PR DESCRIPTION
**Pull request by HoangLanh**

### CODE REVIEW

File: app/assets/stylesheets/custom.css.scss [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+#logo {
+  float: left;
+  margin-right: 10px;
+  font-size: 1.7em;
+  color: #fff;
+  text-transform: uppercase;
+  letter-spacing: -1px;
+  padding-top: 9px;
+  font-weight: bold;
+  }
```
Comment (phanbt58) [inline app/assets/stylesheets/custom.css.scss]: ident

File: app/views/static_pages/about.html.erb [hunk 2]
```diff
@@ -0,0 +1,2 @@
+<% provide :title, t("static_pages.about.title") %>
+  <h1><%= t "static_pages.about.title" %></h1>
```
Comment (phanbt58) [inline app/views/static_pages/about.html.erb]: ident

File: app/views/static_pages/contact.html.erb [hunk 3]
```diff
@@ -0,0 +1,2 @@
+<% provide :title, t("static_pages.contact.title") %>
+  <h1><%= t "static_pages.contact.title" %></h1>
```
Comment (phanbt58) [inline app/views/static_pages/contact.html.erb]: ident

**Comment by phanbt58** [comment]
ok

File: app/views/layouts/application.html.erb [hunk 4]
```diff
@@ -1,14 +1,23 @@
...
-
-<%= yield %>
-
-</body>
+  <head>
+    <%= stylesheet_link_tag "application", media: "all",
+      "data-turbolinks-track": true %>
+    <%= javascript_include_tag "application", "data-turbolinks-track": true %>
+    <%= csrf_meta_tags %>
+      <!--[if lt IE 9]>
```
Comment (dieunb) [inline app/views/layouts/application.html.erb]: indent

**Pull request merged**